### PR TITLE
feat(Colors): Add low opacity brand color variables, add nav blue bra…

### PR DESCRIPTION
**Pull request by iamchristoph**
…nd color variables, add indigo and violet opaque color variables

## Motivation

During work on [this story](https://www.pivotaltracker.com/story/show/182263829) a color was requested that was not available in the design system. This adds that color (--mds-brand-8) as well as some other colors approved my the design team.

## PR upkeep checklist

- [ ] Change log entry
- [ ] Label(s)
- [x] Assignee(s)
- [ ] Deployment URL: https://mavenlink.github.io/design-system/$BRANCH/
- [ ] (When ready for review) Reviewer(s)
- [ ] Green Bigmaven CI check
- [ ] DevQA on Bigmaven staging (e.g. does the work need to be imported in the internal design system?)

### CODE REVIEW

**Review by guanw88 — APPROVED** [review]
🌈

File: src/styles/colors-v2.css [hunk 1]
```diff
@@ -27,6 +37,25 @@
   --mds-grey-opaque-8: #ebebeb;
   --mds-grey-opaque-3: #f7f7f7;
 
+  /* Kantata Branding colors */
```
Comment (juanca) [inline src/styles/colors-v2.css:40]: 🔥

**Comment by juanca** [comment]
I'll cuta release for this.

**Comment by juanca** [comment]
Welcome to merge when it is green.

**Comment by guanw88** [comment]
@iamchristoph I talked with Clarke and some of the `kantata` colors are actually legacy Mavenlink colors that should no longer be used. I'll make a new PR to remove them.

**Comment by guanw88** [comment]
Follow-up PR here: https://github.com/mavenlink/design-system/pull/462

**Pull request merged**